playbooks/roles/clean_puppet/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
ack. Was not sure about the history and thought porting was cheaper then thinking.